kafka_consumer/tests/docker/kerberos/docker-compose.yaml

```diff
@@ -5,6 +5,8 @@ services:
     hostname: kdc.kerberos-demo.local
     build: ./kdc
     container_name: kdc
+    ports:
+      - "88:88"
```

Comment:
Needed to expose the KDC to the Agent and picked port 88 since that seemed to be [default](https://web.mit.edu/kerberos/krb5-1.5/krb5-1.5.4/doc/krb5-install/Ports-for-the-KDC-and-Admin-Services.html) for KDC.